projects/sdk/src/lib/silo.test.ts

```diff
@@ -217,13 +219,55 @@ describe("Deposit Permits", function () {
 
     // Verify
     const allowance = await sdk.contracts.beanstalk.depositAllowance(owner, spender, token);
-    expect(allowance.toString()).to.be.eq(amount);
+    chaiExpect(allowance.toString()).to.be.eq(amount);
   });
 });
 
-///
-// describe('Contract aliases', function () {
-//   it('calls aliased $ methods properly', async () => {
-//     expect(await sdk.silo.$lastUpdate(account1)).to.be.greaterThan(0);
-//   });
-// })
+describe("Silo mowMultiple", () => {
+  const account = account1;
+  const whitelistedToken = sdk.tokens.BEAN;
+  const whitelistedToken2 = sdk.tokens.BEAN_CRV3_LP;
+  const nonWhitelistedToken = sdk.tokens.DAI;
+  const whitelistedTokenAddresses = Array.from(sdk.tokens.siloWhitelist.values()).map((token) => token.address);
+
+  beforeEach(() => {
+    // We mock the methods used in mowMultiple
+    // jest.spyOn(Silo.sdk, 'getAccount').mockResolvedValue(account);
+    // jest.spyOn(Silo.sdk.tokens, 'isWhitelisted').mockImplementation((token) => token === whitelistedToken);
+    // jest.spyOn(Silo.sdk.tokens, 'siloWhitelist').mockReturnValue(new Set([whitelistedToken]));
+    // jest.spyOn(Silo.sdk.contracts.beanstalk, "mowMultiple").mockImplementation(() => "mockedTransaction" as any);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("throws when no tokens provided", async () => {
+    expect(sdk.silo.mowMultiple(account, [])).rejects.toThrow("No tokens provided");
+  });
+
+  it("throws when non-whitelisted token provided", async () => {
+    await expect(sdk.silo.mowMultiple(account, [nonWhitelistedToken])).rejects.toThrow(`${nonWhitelistedToken.symbol} is not whitelisted`);
+  });
+
+  it.skip("warns when single token provided", async () => {
```

Comment:
I'll re-enable this and following tests later on when doing more testing against a local fork with the silo v3 code deployed